src/components/ActivityFeed/ActivityFeedProgressinlisting.tsx

```
@@ -49,12 +49,12 @@ const ActivityFeedProgressinlisting = ({ tally, onchainId, status, proposalType,
 	});
 	const { ayes, nays } = tallyAyeNayVotes;
 
-	const ayeVotesNumber = usingTallyForAyeNayVotes ? ayes : bnToIntBalance(tallyData.ayes || ZERO);
-	const totalVotesNumber = usingTallyForAyeNayVotes ? ayes + nays : bnToIntBalance(tallyData.ayes?.add(tallyData.nays || ZERO) || ZERO);
+	const ayeVotesNumber = usingTallyForAyeNayVotes ? ayes : bnToIntBalance(tallyData?.ayes || ZERO);
+	const totalVotesNumber = usingTallyForAyeNayVotes ? ayes + nays : bnToIntBalance(tallyData?.ayes?.add(tallyData?.nays || ZERO) || ZERO);
 	const ayePercent = totalVotesNumber > 0 ? (ayeVotesNumber / totalVotesNumber) * 100 : 0;
 	const nayPercent = 100 - ayePercent;
-	const isAyeNaN = Number.isNaN(ayePercent);
-	const isNayNaN = Number.isNaN(nayPercent);
+	const isAyeNaN = isNaN(ayePercent);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)